simpleParser.go

```diff
@@ -846,22 +1003,27 @@ func (ctx *expressionParserContext) outputRegex(node ParserTreeNode) (Expression
 	return RegexExpr{node.data}, nil
 }
 
-func (ctx *expressionParserContext) outputField(node ParserTreeNode) (Expression, error) {
+func splitFieldTokensIntoPath(fieldVar string) []string {
```

Comment:
does this method need to handle "[" and "]"?

Reply:
This no longer needed.